src/components/Navbar.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { AppBar, Toolbar, Typography, Button } from '@mui/material';
+import { Link } from 'react-router-dom'; // You can use react-router-dom for handling links
+import Logout from './Logout';
+import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
+
+const Navbar = () => {
+  const { status } = useAuth();
+  const isLoggedIn = status === AuthStatus.LoggedIn;
+  return (
+    <AppBar position="static">
+      <Toolbar>
+        <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+          Music
+        </Typography>
+        <Button color="inherit" component={Link} to="/signIn">
```

Comment:
I don't see a ticket for creating the navbar, so I'm not sure what needs to be implemented in this PR, but I would add conditional rendering to each link in this component based on whether the user is logged in or not. Logged in users should only see the logout and home links, while users who are not logged in should only see the signin and signup links. I see you already have the logout button being conditionally rendered! If there's a reason for not doing that for the rest of the links right now feel free to leave it as is.

Reply:
+1